hippod/garbage_handler.py

```diff
@@ -0,0 +1,195 @@
+                for item in container_garbage:
+                    garbage_list.append(item)
+            return garbage_list
+
+
+        def get_sec(time):
+            time_str = str(time)
+            h, m, s = time_str.split(':')
+            return float(h)*3600 + float(m)*60 + float(s)
+
+
+        def correct_subcontainer(app, sha_major, sha_minor, achiev_id):
+            path = os.path.join(app['DB_OBJECT_PATH'],
+                        sha_major[0:2],
+                        sha_major,
+                        sha_minor,
+                        'subcontainer.db')
+            with open(path) as data_file:
+                data = json.load(data_file)
+                for i, achiev in enumerate(data['achievements']):
+                    if achiev['id'] == achiev_id:
+                        del data['achievements'][i]
+            with open(path, 'w') as data_file:
+                data = json.dumps(data, sort_keys=True,indent=4,
+                      separators=(',', ': '))
+                data_file.write(data)
+
+
+        def correct_container(app, sha_major, sha_minor):
+            path = os.path.join(app['DB_OBJECT_PATH'],
+                        sha_major[0:2],
+                        sha_major,
+                        'container.db')
+            with open(path) as data_file:
+                data = json.load(data_file)
+                for i, sc in enumerate(data['subcontainer-list']):
+                    if sc['sha-minor'] == sha_minor:
+                        del data['subcontainer-list'][i]
+            with open(path, 'w') as data_file:
+                data = json.dumps(data, sort_keys=True,indent=4,
+                      separators=(',', ': '))
+                data_file.write(data)
+
+
+        def correct_object_index(app, sha_major):
+            path = os.path.join(app['DB_OBJECT_PATH'],
+                        'object-index.db')
+            with open(path) as data_file:
+                data = json.load(data_file)
+                for i, d in enumerate(data):
+                    if d['object-item-id'] == sha_major:
+                        del data[i]
+            with open(path, 'w') as data_file:
+                data = json.dumps(data, sort_keys=True,indent=4,
+                      separators=(',', ': '))
+                data_file.write(data)            
+
+
+        def check_achiev_lifetime(self, app, sha_major, sha_minor, sc_content, sc_list):
+            achiev_list = list()
+            for achiev in sc_content['achievements']:
+                achiev_id = str(achiev['id'])
+                date_added = achiev['date-added']
+                achievement = hippod.api_shared.get_achievement_data_by_sha_id(app, sha_major, sha_minor, achiev_id)
+                date_added = datetime.datetime.strptime(date_added, '%Y-%m-%dT%H:%M:%S.%f')
+                diff = datetime.datetime.now() - date_added
+                diff = GarbageHandler.GarbageHandlerCollect.get_sec(diff)
```

Comment:
this looks strange somehow!

```
datetime.datetime.now() - date_added
```

is a timedelta object, right? If so than simple call diff.seconds() to convert to seonds, or miss I something?